modules/sdk-core/src/bitgo/wallet/iWallet.ts

```diff
@@ -114,6 +114,7 @@ export interface PrebuildTransactionOptions {
    * If set to false, sweep all funds including the required minimums for address(es). E.g. Polkadot (DOT) requires 1 DOT minimum.
    */
   keepAlive?: boolean;
+  txFormat?: string;
```

Comment:
This name is quite generic but it seems its just for UTXO now. can you put some comment on this?

Reply:
Should we put a type on this field?

Reply:
Done